flashinfer/fused_moe/core.py

```diff
@@ -846,6 +848,8 @@ def trtllm_gen_fused_moe_sm100_module() -> JitSpec:
         + sm100a_nvcc_flags,
         extra_ldflags=["-lcuda"],
         extra_include_paths=[
+            # link "include" sub-directory in cache
+            jit_env.FLASHINFER_CACHE_DIR / "cubins" / include_path,
```

Comment:
This might be conflicting with https://github.com/flashinfer-ai/flashinfer/pull/1462, considering using "FLASHINFER_CUBIN_DIR" instead